packages/wallet-sdk/package.json

```diff
@@ -43,7 +43,6 @@
     "eventemitter3": "^5.0.1",
```

Comment:
dear reviewers,
recommend to turn on `Hide whitespace`
https://github.com/coinbase/coinbase-wallet-sdk/pull/1018/files?diff=split&w=1
ty!